src/main/resources/application.yml

```diff
@@ -24,4 +24,14 @@ logging:
   pattern:
     dateFormat: yyyy-MM-dd HH:mm:ss.SSS ZZZ
   level:
-    com.courtcanva: debug
\ No newline at end of file
+    com.courtcanva: debug
+
+
+
+management:
+  endpoints:
+    web:
+      cors:
+        allowed-headers: "*"
```

Comment:
Do we really need double quotes here? Since I want it to keep consistency with the way ` allowed-methods` used.